packages/block-library/src/post-comment-author/index.js

```diff
@@ -7,7 +7,7 @@ import edit from './edit';
 /**
  * WordPress dependencies
  */
-import { postAuthor as icon } from '@wordpress/icons';
+import { postCommentsAuthor as icon } from '@wordpress/icons';
```

Comment:
Shouldn’t it be singular form for the comment word as in other places?
```suggestion
import { postCommentAuthor as icon } from '@wordpress/icons';
```

Reply:
As is a new icon, I will remove the post word. For the future "removal" of the word post in every comment related.